test/langtools/jdk/jshell/ToolEnablePreviewTest.java

```diff
@@ -73,19 +73,19 @@ public void testCommandLineFlag() {
     @Test
     public void testCompilerTestFlagEnv() {
         test(new String[] {"-C", "-XDforcePreview"},
-                (a) -> assertCommandOutputContains(a, "Function<Integer,Integer> f = i -> i + i",
+                (a) -> assertCommandOutputContains(a, "Function<Integer,Integer> f = (var i) -> i + i",
```

Comment:
surprised to see these changes

Reply:
similar as comment before `LAMBDA` is being removed but `VAR_SYNTAX_IMPLICIT_LAMBDAS` is still there